src/components/structures/UserSettings.js

```diff
@@ -228,6 +257,7 @@ module.exports = React.createClass({
         }
 
         this._refreshMediaDevices();
+        this._refreshIgnoredUsers();
```

Comment:
I guess technically we might want to also _refreshIgnoreUsers() when we receive an ignoring event from the js-sdk.  But that's such an edge case i'm happy to ignore it for now :)

Reply:
(although given we already have an onAction() here, surely it's just 3 lines to add in a:
```
case 'ignore_state_changed':
                this._refreshIgnoredUsers();
                break;
```
...which almost seems worth it.